Formula/pipenv.rb

```diff
@@ -18,30 +18,11 @@ class Pipenv < Formula
 
   depends_on "python@3.10"
   depends_on "six"
+  depends_on "virtualenv"
```

Comment:
`virtualenv` formula is installed into a virtualenv itself. If `pipenv` uses the library, then it will need help finding the site-packages. Also, the install section adds a symlink to previous installed `virtualenv` resource, which is now broken.